src/index.ts

```diff
@@ -148,9 +153,9 @@ async function login(
         // get access cookie for future access without credentials
         const cookies = <Cookie[]>jar?.toJSON().cookies;
         const code = cookies.find((c) => c.key === config.authCookie);
-        const cards = await accessCards(config, userInfo.userId);
+        const cards = await accessCards(config, userInfo.userId, code?.value);
 
-        if (code && !cards.error) {
+        if (code?.value && !cards.error) {
```

Comment:
Why do we check code.?value here? We can do it before the request (if this check required).

Reply:
Here we send this code to store it in the backend user session. We don't use during login.

Reply:
We can check `code?.value` condition right at the moment after we got it (after 155th line) to avoid extra request in `accessCards`.

Reply:
For now (until we get updates from the NetReady) this might block the login

Reply:
And now, we have updates from netready.